docker-compose.yml

```diff
@@ -30,8 +30,8 @@ services:
       - DATABASE_URL=postgresql://postgres:postgres@db/hmpps-book-secure-move-api
       - SECRET_KEY_BASE=h463472fhaa91f4a277002e9652f86f330e636358e2090d86ab4a4f06845ege1
       - SERVE_API_DOCS=true
-      - AWS_ACCESS_KEY_ID=fakeid
```

Comment:
these keys (`AWS_ACCESS_KEY_ID` and `AWS_SECRET_ACCESS_KEY`) I think are used for document uploads and document retrieval when booking moves (youth moves). They point to different S3 buckets with different credentials to the bucket for storing feeds... so unfortunately I think you'll need to support access to both buckets simultaneously

Reply:
I thought that'd be the case too, but I couldn't find `AWS_ACCESS_KEY_ID` anywhere else in this project, any ideas?

Reply:
its probably set in the deploy repo... I'll see if I can find it

Reply:
Also these didn't match up, the AWS_ACCESS_KEY_ID was the reporting access key, but we have S3_BUCKET_NAME defined as the documents bucket in here, a bit confusing :D

Reply:
Take a look at `app/lib/cloud_data/reports_feed.rb` vs `app/lib/cloud_data/metrics_feed.rb`
The reports feed assumes the default AWS env vars; whereas the metrics feed explicitly names them:

```
 @client = client || Aws::S3::Client.new(
        region: ENV['S3_METRICS_REGION'],
        access_key_id: ENV['S3_METRICS_ACCESS_KEY_ID'],
        secret_access_key: ENV['S3_METRICS_SECRET_ACCESS_KEY'],
      )
```

to avoid confusion and naming clashing lets use the env var names you have in your [deploy PR](https://github.com/ministryofjustice/hmpps-book-secure-move-api-deploy/pull/150/files#diff-2029cf907f6ef6b89c9bcaf88b53791aa6899564eff891b20a94c6920e1ba724R116), i.e.:
* `S3_REPORTING_ACCESS_KEY_ID`
* `S3_REPORTING_SECRET_ACCESS_KEY`
* `S3_REPORTING_BUCKET_NAME`

- so could you update `app/lib/cloud_data/reports_feed.rb` to use these new names ^ ?

Reply:
Sounds like a good idea to me, I'm on it